### DISCUSSION TITLE
Support for manifest v3 for chrome extension

### DISCUSSION THREAD

**Discussion by ajaxe** [quasarframework/quasar#8844] [CLI - BEX mode] [answered]
I like the developer experience using the BEX cli, but, I would like to know if there are plans to support newer manifest v3 for chrome extensions.

If it is on the roadmap perhaps an indication of when this support is slated for a release would be good to know.

**Comment by webnoob** [comment]
I need to review the changes for bringing BEX inline with v3. I'll do this ASAP.

**Comment by ajaxe** [comment, marked as answer]
Thanks for the support of manifest v3

**Reply by rstoenescu** [reply]
Yes, I implemented it for Quasar CLI with Vite (only) because it entails some breaking changes to the BEX folder structure (most notably, but not the only thing: manifest v3 requires the background script to be at the root level of the extension). Rather than breaking everyone's q/app-webpack projects, I went with implementing the future of BEX into q/app-vite only.
